specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2017-10-01/containerregistry.json

```diff
@@ -1014,6 +1055,48 @@
     }
   },
   "definitions": {
+    "ImageCopyFromModel": {
+      "required": [
+        "sourceRepository",
+        "sourceRegistryResourceId"
+      ],
+      "properties": {
+        "sourceRepository": {
+          "description": "Repository name of the source image.",
+          "type": "string"
+        },
+        "sourceTag": {
+          "description": "The tag name of the source image.  When both source tag and source manifest are omitted the 'latest' tag will be used.\r\nExclusive with SourceManifestDigest.",
+          "type": "string"
+        },
+        "sourceManifestDigest": {
+          "description": "The manifest sha of the source image. Exclusive with SourceTag.",
+          "type": "string"
+        },
+        "sourceRegistryResourceId": {
+          "description": "The resource id of the source registry.",
+          "type": "string"
+        },
+        "targetTags": {
+          "description": "List of strings of the form repo[:tag].  When tag is omitted the source will be used (or 'latest' if source tag is also omitted.)",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "untaggedTargetRepositories": {
+          "description": "List of strings of repository names to do a manifest only copy.  No tag will be created.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "force": {
```

Comment:
can we make this a string enum? ex - "copyMode" propetrties where values are force etc.

Reply:
Created a new mode enum.